### PR TITLE
Test #12661 Change page title after editing a host

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto roidelapluie@inuits.eu

### CODE REVIEW

**Comment by roidelapluie** [comment]
Actually, not the fix but the test

**Comment by timogoebel** [comment]
@roidelapluie : Thanks for the test. The commit message however must start with Fixes #xxxx. If your fix extends a previous fix that has not been shipped in a release, use Refs #xxxx.

File: test/integration/host_test.rb [hunk 1]
```diff
@@ -130,4 +130,14 @@ class HostIntegrationTest < ActionDispatch::IntegrationTest
       assert page.has_no_selector?('#params .has-error')
     end
   end
+
+  describe 'edit and view host page' do
+    test 'titles aret set properly when switching from Edit to View host' do
```
Comment (timogoebel) [inline test/integration/host_test.rb:135]: Nitpick: typo - "aret" should be "are"

**Comment by roidelapluie** [comment]
@timogoebel I am trying to write a test to reproduce #12661 before I fix it -- but the test passes (but it should fail)

**Pull request closed without merging**